seqr/views/apis/report_api.py

```diff
@@ -1130,9 +1165,13 @@ def _validate_column_data(column, file_name, data, column_validator, warnings, e
                 check_recommend_condition = WARN_MISSING_CONDITIONAL_COLUMNS.get(column)
                 if not check_recommend_condition or check_recommend_condition(row):
                     warn_missing.append(_get_row_id(row))
-        elif enum and value not in enum:
+        elif data_type_validator and not data_type_validator(value, column_validator):
```

Comment:
But it is neccessary because not all fields have a data type valuator, and if a field does have one we need to check it. The line you linked to also checks for unique or required validation, so it might have those but not data_type_validator